scene/gui/rich_text_label.h

```diff
@@ -643,6 +644,14 @@ class RichTextLabel : public Control {
 	String _roman(int p_num, bool p_capitalize) const;
 	String _letters(int p_num, bool p_capitalize) const;
 
+	static Rect2i _merge_or_copy(const Rect2i &p_a, const Rect2i &p_b) {
```

Comment:
We usually put helper method like this in the cpp file (i.e. they aren't in header at all).